client/my-sites/upgrades/checkout/purchase-detail.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+
+const PurchaseDetail = ( { additionalClass, buttonText, description, onButtonClick, title } ) => (
```

Comment:
Good catch - I updated the second commit to assign `propTypes` for this component.